src/test/java/eu/europeana/metis/sandbox/service/dataset/AsyncDatasetPublishServiceImplTest.java

```diff
@@ -223,13 +212,8 @@ void runHarvestOaiAsync_withoutXsltReachMaxRecords_expectSuccess() throws Harves
 
   @Test
   void runHarvestOaiAsync_withXslt_expectSuccess() throws HarvesterException {
-    Record recordData = Record.builder()
-            .country(Country.NETHERLANDS)
-            .language(Language.NL)
-            .datasetName("datasetName")
-            .datasetId("datasetId")
-            .content(new byte[0])
-            .build();
+    Record recordData = Record.builder().country(Country.NETHERLANDS).language(Language.NL).datasetName("datasetName")
+        .datasetId("datasetId").content(new byte[0]).build();
 
```

Comment:
this formatting was easy to read. :stuck_out_tongue_winking_eye:  if you can format all the ones that use this builder  like this I think it will be easy to read.
```
Record recordData = Record.builder()
            .country(Country.NETHERLANDS)
            .language(Language.NL)
            .datasetName("datasetName")
            .datasetId("datasetId")
            .content(new byte[0])
            .build();
```
because your eyes will do a down scan, instead of a side scan when reading.

Reply:
Wrong formatter. I still don't know why but intellij sometimes just defaults to GoogleStyle Formatter. But the metis-framework formatter also doesn't respect these kind of line breaks, it puts it all in one line. The code is now in several lines, more readable as it should